lib/govspeak.rb

```diff
@@ -305,7 +305,7 @@ def render_image(image)
       <<~BODY
 
         <div class="address"><div class="adr org fn"><p markdown="1">
-        #{body.sub("\n", '').gsub("\n", '<br />')}
+        #{body.sub(/\r?\n/, '').gsub(/\\*\r?\n/, '<br />')}
```

Comment:
**nitpick:** slash regex

It has to be a multiple of two trailing backslashes (the documentation says 2+, but my testing suggested it needs to be a multiple of two), so you could do `(?:\\\\)*`, but I can't see why anyone would be leaving random backslashes on the ends of address block lines anyway

Reply:
Sorry, this is another small thing I should've mentioned. I couldn't get odd numbers of backslashes to play nice. They didn't just end up in the output HTML as is, they broke the encoding of the HTML line break 😓 That's why I went for the nuclear option